rai/client.go

```diff
@@ -1720,22 +1720,24 @@ func (c *Client) ListSnowflakeDatabaseLinks(
 //
 
 type DataStreamOpts struct {
-	IsView      bool
 	RaiDatabase string
 	Relation    string
 	ObjectName  string
+	ObjectType  string
```

Comment:
are there any object types other than table and view? Can the ObjectType be an enum?

Reply:
not for now, only table and view

Patrick commented, comparing against the create engine with takes a string arg for engine size: "... the service is the authority on the allowed values

also true in our case I suppose

there's a tension in contracts that's annoying where an enumerated list in the contract means additions is backward-compatibility breaking if the client enforces it, that's the main driver behind that design "